_scripts/instruction-widget/templates/getting-started/nginx.html

```diff
@@ -1,17 +1,33 @@
-{{> certonly}}
+<p>
+Certbot has an alpha-quality Nginx plugin, which is supported on
+many platforms, and automates both obtaining and installing certs:
+</p>
+<pre>$ {{base_command}} --nginx</pre>
+<p>
+If you're feeling more conservative and would like to make the changes to your
```

Comment:
I'd add a sentence saying something like:
"this will get a certificate for you and have Certbot edit your nginx config automatically to serve it"